From 488ae41be478e2f42832f2ced851ac846807314d Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Mon, 16 Oct 2023 12:23:11 +0200
Subject: [PATCH 0008/1695] core: kernel: add missing guards in time_source.h

Adds missing guards in header file time_source.h.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 5c3b174fdb2f2dbb618b196f9d065cacfc898e3a core: kernel: add missing guards in time_source.h]
Change-Id: Icd317b6c46b3c9ea76ae7b9ae333c2a05c6461f8
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from 5d7e3db242df88a3c12e762161718bf1bd5cea79]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/include/kernel/time_source.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/core/include/kernel/time_source.h b/core/include/kernel/time_source.h
index fce5391a0..e02f98432 100644
--- a/core/include/kernel/time_source.h
+++ b/core/include/kernel/time_source.h
@@ -2,6 +2,8 @@
 /*
  * Copyright (c) 2014, Linaro Limited
  */
+#ifndef __KERNEL_TIME_SOURCE_H
+#define __KERNEL_TIME_SOURCE_H
 
 #include <kernel/tee_time.h>
 
@@ -18,3 +20,4 @@ void time_source_init(void);
 	}
 
 extern struct time_source _time_source;
+#endif /*__KERNEL_TIME_SOURCE_H*/
-- 
2.47.2

